functests/1_performance/performance.go

```diff
@@ -140,7 +140,7 @@ var _ = Describe("[rfe_id:27368][performance]", func() {
 
 	Context("Tuned CRs generated from profile", func() {
 		tunedExpectedName := components.GetComponentName(testutils.PerformanceProfileName, components.ProfileNamePerformance)
-		It("[test_id:31748] Should have the expected name for tuned from the profile owner object", func() {
+		FIt("[test_id:31748] Should have the expected name for tuned from the profile owner object", func() {
```

Comment:
Another focus that should not be merged.

Reply:
Yep its a WIP just to show you the code for my questions, I will clean before delete WIP tag